docs/static/Microsoft_Makecode.svg

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We have plenty of icons under /static/icons . Is this one new?

Reply:
Yes, this is a new one. But now that I see it I think its place in under the docs/static/logo folder. We will move it there.

Reply:
Stick to “-“ instead of _ in files names. Avoid capital letters too (Linux vs windows quirks)